src/hosts/host.entity.mjs

```diff
@@ -2,7 +2,8 @@
 import * as net from 'node:net'
 import * as dgram from 'node:dgram'
 /* eslint-enable */
-import { nanoid } from 'nanoid'
+import { nanoid, customAlphabet } from 'nanoid'
```

Comment:
Please change to `import * as nanoid from 'nanoid'`, I'd prefer to have it apparent in the code that it's a nanoid :slightly_smiling_face: